lib/src/TimePicker/TimePickerToolbar.tsx

```diff
@@ -139,33 +133,34 @@ export const TimePickerToolbar: React.FC<ToolbarComponentProps> = ({
 
         {arrayIncludes(views, 'seconds') && (
           <ToolbarButton
-            variant="h2"
-            onClick={() => setOpenView(ClockType.SECONDS)}
-            selected={openView === ClockType.SECONDS}
+            variant={clockTypographyVariant}
+            onClick={() => setOpenView('seconds')}
+            selected={openView === 'seconds'}
             label={utils.getSecondText(date)}
           />
         )}
       </div>
 
-      {ampm && (
+      {showAmPmControl && (
         <div
           className={clsx(classes.ampmSelection, {
             [classes.ampmLandscape]: isLandscape,
-            [classes.ampmSelectionWithSeconds]: arrayIncludes(views, 'seconds'),
           })}
         >
           <ToolbarButton
+            data-test-id="toolbar-am-btn"
```

Comment:
For reference, we have a `data-mui-test` convention in the main repository to solve this problem (+ a babel plugin to remove it before releasing). It would be great to unify the approach.

@eps1lon should we rename `data-mui-test` to `data-testid`, to match @testing-library default behavior?
```suggestion
            data-testid="toolbar-am-btn"
```

Reply:
Actually I don't like @testing-library so now working on improving cypress with component tests https://github.com/cypress-io/cypress/pull/5923. 
Going to try use it with pickers.
> P.S `data-testid` is incorrect word for spell checkers, but I don't care about it

Reply:
So, what do you think that we should do here? `data-testid` seems to match https://docs.cypress.io/guides/references/best-practices.html.

Reply:
Hehe, my bad :DDDD will rename it